consul/Module.md

```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
 
 Register services in Consul with the given `jsonPayload`.
 ```ballerina
+json jsonPayload = {"ID":"redis", "Name":"redis1", "Address":"localhost", "port":8000, "EnableTagOverride":false};
 var serviceRegister = consulClient->registerService(jsonPayload);
 if (serviceRegister is boolean) {
-   io:println(serviceRegister);
+   io:println("Status of the service: ", serviceRegister);
 } else {
-   io:println(<string>serviceRegister.detail().message);
+   io:println("Error: ", <string>serviceRegister.detail().message);
 }
 ```
 
 Get the details of the service with the given `serviceName`.
 ```ballerina
 var serviceDetails = consulClient->getService(serviceName);
 if (serviceDetails is consul:CatalogService[]) {
-   io:println(serviceDetails);
+   io:println("Details of a particular service: ", serviceDetails);
 } else {
-   io:println(<string>serviceDetails.detail().message);
+   io:println("Error: ", <string>serviceDetails.detail().message);
 }
 ```
 
 Register a check in Consul with the given `jsonCheck`.
 ```ballerina
 var checkRegister = consulClient->registerCheck(jsonCheck);
 if (checkRegister is boolean) {
-   io:println(checkRegister);
+   io:println("Status of the check: ", checkRegister);
```

Comment:
```suggestion
   io:println("Status of the check registration: ", checkRegister);
```

We may also have to rename the variable to something like `checkRegistrationResponse`